### DISCUSSION TITLE
Welcome to gr-satellites discussions

### DISCUSSION THREAD

**Discussion by daniestevez** [daniestevez/gr-satellites#304] [Announcements]
A few months ago, Github [has added support for discussions](https://github.blog/2021-08-17-github-discussions-out-of-beta/) in repositories.

Moving forward, gr-satellites will use the [discussions](https://github.com/daniestevez/gr-satellites/discussions) for the following:
* Project related [announcements](https://github.com/daniestevez/gr-satellites/discussions/categories/announcements)
* All sorts of [general discussion](https://github.com/daniestevez/gr-satellites/discussions/categories/general) regarding gr-satellites. Since gr-satellites doesn't have any mailing list or forum, I think this will be a good place for the community to gather.
* [Ideas for improvement](https://github.com/daniestevez/gr-satellites/discussions/categories/ideas), new features, etc.
* Any sort of [questions](https://github.com/daniestevez/gr-satellites/discussions/categories/q-a) about the use of gr-satellites or related topics. This represents a large amount of the issues that have been opened so far, but the discussions page seems a better frame for this.
* [Cool projects using gr-satellites](https://github.com/daniestevez/gr-satellites/discussions/categories/show-and-tell). Github includes the "Show and tell" category by default, and it seems interesting enough to leave it in.
* [Satellite teams](https://github.com/daniestevez/gr-satellites/discussions/categories/satellite-teams). So far satellites teams were using the issues to discuss new features or support required for their mission, but it seems better to have a dedicated discussion category.

The [issues](https://github.com/daniestevez/gr-satellites/issues) will be used for the following:
* Bug reports
* Technical problems, such as installation problems
* Feature requests or ideas for enhancements. While this may have some overlap with the "ideas" discussions, I think that it's best to keep the issues for feature requests that are more concrete and to track the progress of development of new features, and use the issues, and the discussions for more vague ideas.
* Anything else that deals with the code more directly than the topics in the discussions.

**TLDR:** moving forward the idea is to use discussion for general talk and to keep the issues for matters that are more related to the code itself (and often will have associated commits or pull requests).

I hope that this will improve the way that the users of gr-satellites will interact among themselves and with the project, but only time will tell if this idea turns out to be practical. Let me know your thoughts.

**Comment by Osama-Moustafa** [comment]
I am developing a satellite communications testbed that uses CCSDS protocols for data transmission. I am working on a real-time transmission system using GNU Radio and Software Defined Radios (bladeRF x40), and I am encountering persistent issues with decoding CCSDS-encoded data using the recommended format.
 
Despite following the CCSDS standard gr-satellites (https://github.com/daniestevez/gr-satellites.git) blocks on GNU Radio  (via BPSK modulation and demodulation), I didn't receive any output, even though we don't have any hardware errors.  (Please find the attached screenshots of the used flowcharts).

![RECEIVER--](https://github.com/user-attachments/assets/f64a15c3-f900-4452-b30c-535d7be08ac5)
![SENDER--](https://github.com/user-attachments/assets/51796352-9639-42d9-9c15-876ea2262df5)

**Reply by daniestevez** [reply]
Can you please move this off to a new discussion?
